modules/mobile/spec/workers/vet360_linking_spec.rb

```diff
@@ -7,13 +7,25 @@
   let(:user) { create(:user, :loa3) }
 
   context 'when linking request is successfully made' do
-    it 'logs the user id and transaction id that linked an account with vet360' do
+    it 'logs the user id, transaction id, and creates a mobile user that linked an account with vet360' do
       VCR.use_cassette('mobile/profile/init_vet360_id_success') do
         expect(Rails.logger).to receive(:info).with(
           'Mobile Vet360 account linking request succeeded for user with uuid',
           { user_uuid: user.uuid, transaction_id: 'd8951c96-5b8c-42ea-9fbe-e656941b7236' }
         )
         subject.perform(user.uuid)
+        expect(Mobile::User.where(icn: user.icn)).to exist if Mobile::User.can_count_attempts?
+      end
+    end
+
+    if Mobile::User.can_count_attempts?
```

Comment:
Madness, I tell you. If this is the rule, it's a bad rule that shouldn't apply here because there was no model prior to this. I haven't seen any code like this elsewhere in the app so I'm doubtful this is the intent.